### ISSUE TITLE
set up proxy for neo4j requests

### ISSUE THREAD

**Issue by xiurobert** [TwigSlot/twig#3]
there's not much that can be done on the client side before the neo4j proxy is setup. 

we can use Flask, and for now the basic API can just be: the client sends a GET request with a cypher query that the server forwards to neo4j. 

thanks!

**Comment by tch1001** [comment]
Hmm but I want people to be able to view the graph without logging in, just that they can only save if they log in. Perhaps we can have a read limit for the number of rows implemented in flask (just appending LIMIT 1000 in the cypher query should work). Nginx can handle malicious spam requests

**Comment by xiurobert** [comment]
If you want them to be able to view the graph without logging in we can't allow them to execute queries on the neo4j server arbitrarily since actually finding out what that code does is NP-hard (or probably even impossible)

**Comment by tch1001** [comment]
Right... ok guess we can expose an API to the client, and form the queries on server side. For a simple test let's do something like 
/:username -> MATCH (n:User) WHERE n.username = username RETURN n 
I'll implement more routes when I work on client side
